.github/label-commenter-config.yml

```diff
@@ -0,0 +1,15 @@
+labels:
+  - name: needs-docs
+    labeled:
+      pr:
+        body: |
+          Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+          - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
+          - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
+          - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+        action: open
+    unlabeled:
+      issue:
+        body: |
+          Thank you for adding module documentation :tada:
+        action: open
```

Comment:
This will be expanded upon in the future, just putting up an initial PR to see what the thoughts are :+1:

The configuration file can do other things too, like additionally closing pull requests/issues if labeled with `duplicate`, or `termux`, etc, it should work for whatever our workflow is

Label commenter action docs: https://github.com/marketplace/actions/label-commenter